tensorflow_datasets/.vscode/settings.json

```diff
@@ -19,11 +19,10 @@
     },
     "python.formatting.provider": "yapf",
     "python.testing.pytestArgs": [
-        "tensorflow_datasets",
         "-vv",
         "--continue-on-collection-errors",
-        "--ignore=tensorflow_datasets/testing/test_utils.py",
-        "--ignore=tensorflow_datasets/scripts/documentation/build_api_docs_test.py"
+        "--ignore=testing/test_utils.py",
+        "--ignore=scripts/documentation/build_api_docs_test.py"
     ],
     "files.watcherExclude": {
         "**/.git/**": true,
```

Comment:
As `.git/` and `docs/` files are in the `datasets` folder, so maybe we can remove here.